internal/consumer/ackMgr.go

```diff
@@ -195,3 +199,24 @@ func (l *threadSafeSortedList) Add(value int64) (list.Address, error) {
 	}
 	return addr, err
 }
+
+func (l *threadSafeSortedList) Reset() <-chan struct{} {
+	doneC := make(chan struct{})
+	checkInterval := time.NewTicker(time.Millisecond)
```

Comment:
this is too aggressive, can increase cpu usage due to spinning

Reply:
Increased to 1 second.